docs/data/charts/zoom-and-pan/zoom-and-pan.md

```diff
@@ -131,3 +131,75 @@ The `zoom` state is an array of objects that define the zoom state for each axis
 To synchronize zoom between multiple charts, you can control the zoom state.
 
 {{"demo": "ZoomControlled.js"}}
+
+## Zoom configuration
+
+You can have fine-grained control over which interactions are enabled and under which conditions by using the `zoomConfig` prop.
+
+### Interactions
+
+The `zoomConfig` prop allows you to specify which interactions are enabled for zooming and panning:
+
+```jsx
+<BarChartPro
+  zoomConfig={{
+    // Enable both wheel and pinch zoom
+    zoom: ['onWheel', 'onPinch'],
+    // Enable drag panning
+    pan: ['onDrag'],
+  }}
+/>
+```
+
+### Key modifiers
+
+Some interactions allow setting up required keys to be pressed to enable the interaction.
+This can be set up using the `keys` property in the interaction configuration.
+
+```jsx
+<BarChartPro
+  zoomConfig={{
+    // Only zoom when Control key is pressed
+    zoom: [{ type: 'onWheel', keys: ['Control'] }],
+    // Only pan when Shift key is pressed
+    pan: [{ type: 'onDrag', keys: ['Shift'] }],
+  }}
+/>
+```
+
+Available keys include:
+
+- Modifier keys: `'Shift'`, `'Control'`, `'Alt'`, `'Meta'`
+- Special combinations: `'ControlOrMeta'` (useful for cross-platform support)
+- Letter keys: `'a'` through `'z'`
+
+It is also possible to require multiple keys to be pressed simultaneously:
+
+```jsx
+<BarChartPro
+  zoomConfig={{
+    // Only pan when both Shift and Control are pressed
+    pan: [{ type: 'onDrag', keys: ['Shift', 'Control'] }],
+  }}
+/>
+```
+
+### Modes
+
+Interactions can also be restricted to specific pointer types by using the `mode` property:
+
+```jsx
+<BarChartPro
+  zoomConfig={{
+    // Only pan with touch, not mouse
+    pan: [{ type: 'onDrag', mode: 'touch' }],
+  }}
+  // other props
+/>
+```
+
+Available modes:
```

Comment:
We're calling these "modes", but would it make sense to keep the same nomenclature as the web and call it `pointerType`?

Reply:
I've renamed it to `pointerMode` to follow the gesture lib, IMO `pointerType` describes the type, not what the prop does. hopefully `pointerMode` is clearer that it is a pointer, and the mode in which it uses. Alternatively we could rename to `requiredPointerType` though that is a bit too long 😆